assets/common/js/models/Canvas.js

```diff
@@ -1,5 +1,5 @@
 function Canvas() {
-    this.backgroundColor = "#323232";
+    this.backgroundColor = "transparent";
```

Comment:
I checked the file and you don't use this.backgroundColor, is there any reason to keep this variable? If not, you should delete it 😃

Reply:
Good call!